src/patient/PatientInfoFields.tsx

```diff
@@ -0,0 +1,84 @@
+import { Field, ErrorMessage, FormikState } from 'formik';
+import { differenceInYears } from 'date-fns';
+
+export interface PatientInfo {
+  name: string;
+  dateOfBirth: string;
+  eye: 'left' | 'right';
+}
+
+const calcAge = (dateOfBirth: string | undefined, error: string | undefined) =>
```

Comment:
I don't like the `string |  undefined`. Is there a more elegant way to do this?

Reply:
named parameters/defaults